canonical-web-and-design/jaas.ai
# Add /case-studies blog setup

## Done

Add /case-studies blog setup pulling from Ubuntu design blog feed

## QA

- Pull branch
- Run `./run clean && ./run serve`
- View case studies at http://0.0.0.0:8029/case-studies
- Check everything works as expected

Fixes #375, Fixes #376, Fixes #377,
